addons/mail/models/mail_message.py

```diff
@@ -988,11 +995,20 @@ def _message_format(self, fnames):
                         'field_type': tracking.field_type,
                     })
 
+            if message_sudo.model and message_sudo.res_id:
+                record_name = self.env[message_sudo.model]\
+                                  .browse(message_sudo.res_id)\
+                                  .with_prefetch(thread_ids_by_model_name[message_sudo.model].ids)\
+                                  .display_name
```

Comment:
I don't think we have an actual guideline to enforce one or the other, but I really dislike multi-line that are visually aligned, so much lost space on the left... Would be better to just indent once every new line.

Also we usually have a space before the `\` in that case just to make sure the new line split is well separated from the instruction, it makes it a bit easier to read.

Reply:
It's funny to see how people like to read code. I like the complete opposite, I like the "dot aligned" function chain because it's make easier to see all the steps and the order of execution. The dot's act like a ruler you can follow. The extra space on the left isn't "lost" to me (what ever lost might mean to you).
Still, I edited it the way you like :)